test/sass/typograpy.spec.scss

```diff
@@ -159,3 +164,118 @@
   }
 
 }
+
+@include describe('@mixin label-as-content') {
+
+  @include it('generates defaults for label-as-content') {
```

Comment:
Don't need to say "for label-as-content" here, as this `it` clause is already inside the`describe('@mixin label-as-content') ` clause.

Suggest the slightly more descriptive `it('generates correct default styles')`.

Reply:
Not done.